src/main/java/org/scalameter/japi/annotation/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.japi.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/** Annotates benchmark snippet.
+ *
+ *  Its value sets the scope.
```

Comment:
Ups, that was my neglect (and lack of the ScalaDoc comment completion in Java files in my IDE).